bazel/thirdparty/seastar.BUILD

```diff
@@ -634,12 +620,12 @@ cc_library(
         "@boost//:filesystem",
```

Comment:
Looking a little closer at these, is `@boost//:asio` a real dependency for seastar?

Reply:
```
src/net/net.cc:        ip = boost::asio::ip::address_v4::from_string(addr).to_ulong();
```

Reply:
Ah thanks